### PR TITLE
splash screen [branch in progress]

### PR DESCRIPTION
**Pull request by Eliasadam**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: mobile/src/screens/SplashScreen/SplashScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import styled from '@emotion/native';
+// import { Image} from 'react-native';
+import { Layout, Title } from '../../styles/components';
+import { font } from '../../styles/constants';
```
Comment (klbennett) [inline mobile/src/screens/SplashScreen/SplashScreen.jsx:5]: Also need to import 'Text'

File: mobile/src/screens/SplashScreen/SplashScreen.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const Logo = styled.Image`
+  width: 30em;
+  height: 30em;
+`;
+
+const SplashScreen = () => {
+
```
Comment (klbennett) [inline mobile/src/screens/SplashScreen/SplashScreen.jsx:13]: Need to add 'navigation' as an argument `const SplashScreen = ({ navigation }) => {
`

File: mobile/src/screens/SplashScreen/SplashScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import styled from '@emotion/native';
+// import { Image} from 'react-native';
```
Comment (ndeasy) [inline mobile/src/screens/SplashScreen/SplashScreen.jsx:3]: don't commit commented out code

File: mobile/yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,8515 @@
+Arguments:
```
Comment (ndeasy) [inline mobile/yarn-error.log:1]: Don't commit this file. Add it to your git ignore

File: mobile/.idea/mobile.iml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (ndeasy) [inline mobile/.idea/mobile.iml:9]: Add all files under the .idea folder to git ignore

File: .idea/amputee-rehab-app-monorepo.iml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (ndeasy) [inline .idea/amputee-rehab-app-monorepo.iml:9]: Add all files under the .idea folder to git ignore and commit the ignore file. We shouldn't have any .idea folder in git, it's fine to keep it locally on your machine as it's essential project settings specific to your machine

File: package.json [hunk 7]
```diff
@@ -11,14 +11,20 @@
...
     "@commitlint/config-conventional": "^8.3.4",
     "commitizen": "^4.1.2",
     "cz-conventional-changelog": "^3.2.0",
-    "husky": "^4.2.5"
+    "husky": "^4.2.5",
+    "jest": "^26.1.0",
+    "jest-expo": "^38.0.2",
+    "jest-fetch-mock": "^3.0.3",
+    "react-native-testing-library": "^2.1.1",
+    "react-test-renderer": "^16.13.1"
```
Comment (ndeasy) [inline package.json:27]: Are these imports needed? I see they've been added but can't see where they're being used

**Review by ndeasy — APPROVED** [review]

**Pull request closed without merging**